pkg/tsdb/grafana-pyroscope-datasource/instance.go

```diff
@@ -48,15 +49,9 @@ func NewPhlareDatasource(httpClientProvider httpclient.Provider, settings backen
 		return nil, err
 	}
 
-	var jsonData *JsonData
-	err = json.Unmarshal(settings.JSONData, &jsonData)
-	if err != nil {
-		return nil, err
-	}
-
 	return &PhlareDatasource{
 		httpClient: httpClient,
-		client:     getClient(jsonData.BackendType, httpClient, settings.URL),
+		client:     NewPhlareClient(httpClient, settings.URL),
```

Comment:
I'm a bit confused as to why we're still calling this Phlare. Is not everything supposed to be called Grafana-Pyroscope now?

Reply:
Yes but don't want to make this diff convoluted again, will rename in separate PR.